docs/content/mobile/android/local-deploy.md

```diff
@@ -17,6 +17,9 @@ To connect your Android device to a local deployment, you need to prepare its co
 1. Select **Save** and close the window.
 
 ## Unity Editor{#in-editor}
+1. If your project does not already contain one, create a MonoBehaviour script which inherits from the [`MobileWorkerConnector`](https://github.com/spatialos/gdk-for-unity/blob/master/workers/unity/Packages/com.improbable.gdk.mobile/Worker/MobileWorkerConnector.cs) and include the functionality you want. <br> 
+In your Unity Editor, add this script it to your Android client-worker GameObject. In the FPS Starter Project this is located at **Assets** > **Fps** > **Prefabs** > **AndroidClientWorker**.
+1. In your Unity Editor, in the Inspector window, in the script’s drop-down window, ensure that the checkbox `ShouldConnectLocally` is checked.
```

Comment:
this part reads weird as we have three location statements `In your Unity Editor, in the Inspector window, in the script’s drop-down window,` @ElleEss  any suggestions?

Reply:
I reworded this a little bit to the following:
1. In the Unity Editor, navigate to your Android client-worker GameObject and ensure the `ShouldConnectLocally` checkbox is checked in the script's drop-down window of the Inspector window.